webrender_api/src/api.rs

```diff
@@ -872,11 +872,11 @@ pub struct DynamicProperties {
     pub floats: Vec<PropertyValue<f32>>,
 }
 
-pub trait RenderNotifier: Send {
-    fn new_frame_ready(&mut self);
-    fn new_scroll_frame_ready(&mut self, composite_needed: bool);
-    fn external_event(&mut self, _evt: ExternalEvent) {
+pub trait RenderNotifier: Send + Clone {
```

Comment:
would it make sense to provide an empty implementation, e.g. for `()`?

Reply:
I don't see the purpose if no code makes use of it.